lib/elixir/lib/record.ex

```diff
@@ -36,6 +36,10 @@ defmodule Record do
 
   """
 
+  @type type :: :defrecord | :defrecordp
+  @type name :: atom
+  @type tag :: atom
```

Comment:
Sorry, but just because it is not used in the Record module, it doesn't mean we shouldn't support it. Any other library can make use of it. If we are going to support `name` we should support `tag` IMHO.